react/components/UploadModal/ReportInformationDetails.tsx

```diff
@@ -0,0 +1,47 @@
+import { Text, csx } from '@vtex/admin-ui'
+import React from 'react'
+
+import { useTranslate } from '../../hooks'
+
+export type ReportInformationDetailsProps = {
+  variant: 'Import' | 'Upload'
+}
+
+const ReportInformationDetails = ({
+  variant,
+}: ReportInformationDetailsProps) => {
+  const { translate: t } = useTranslate()
+
+  return (
+    <>
+      <Text
+        variant="body"
+        className={csx({
+          display: 'block',
+          marginTop: '$space-3',
+          marginBottom: '$space-2',
+        })}
+      >
+        {t(`reportInformationInitial${variant}`, {
+          a: (content: string) => <a href="/">{content}</a>,
+        })}
+      </Text>
+      <Text variant="body">
+        {t('reportInformationNextSteps')}
+        <ul className={csx({ marginLeft: '$space-1' })}>
+          <ol>{t('reportInformationStep1')}</ol>
+          <ol>{t('reportInformationStep2')}</ol>
+          <ol>{t(`reportInformationStep3${variant}`)}</ol>
+          <ol>
+            {t('reportInformationStep4', {
+              a: (content: string) => <a href="/">{content}</a>,
+              ab: (content: string) => <a href="/">{content}</a>,
```

Comment:
Why this `ab`? It's the same as `a`, right?

Reply:
I did this because theres no way to differ the booth tags, so I called the second tag as ab, but i will change to improve the semantic